arbre.gemspec

```
@@ -21,6 +21,6 @@ Gem::Specification.new do |s|
 
   s.required_ruby_version = '>= 2.5'
 
-  s.add_dependency("activesupport", ">= 3.0.0", "< 6.2")
+  s.add_dependency("activesupport", ">= 3.0.0", "< 7.0")
```

Now that Rails 7 is released, this should probably be changed to:

```suggestion
  s.add_dependency("activesupport", ">= 3.0.0", "< 8.0")
```